src/txn/txn.c

```diff
@@ -1468,11 +1468,13 @@ __wt_txn_stats_update(WT_SESSION_IMPL *session)
       checkpoint_pinned == WT_TXN_NONE ? 0 : txn_global->current - checkpoint_pinned);
 
     WT_STAT_SET(session, stats, txn_checkpoint_prep_max, conn->ckpt_prep_max);
-    WT_STAT_SET(session, stats, txn_checkpoint_prep_min, conn->ckpt_prep_min);
+    if (conn->ckpt_prep_min != UINT64_MAX)
```

Comment:
I think these two checks are redundant because we must have finished a checkpoint to reach here. Keep it to prevent we show something odd.

Reply:
I think it is better to have the check so that even if is called before the first checkpoint in future also, this works